inst/stan/functions/observation_model.stan

```diff
@@ -66,12 +66,12 @@ void truncation_lp(real[] truncation_mean, real[] truncation_sd,
 }
 // update log density for reported cases
 void report_lp(int[] cases, vector reports,
-               real[] rep_phi, int phi_prior,
+               real[] rep_phi, int phi_mean, int phi_sd,
```

Comment:
Shouldn't `phi_mean` and `phi_sd` be `real`?

Reply:
Yes good spot (and thanks for the review)